src/common/Browser.ts

```diff
@@ -650,7 +650,7 @@ export class BrowserContext extends EventEmitter {
    */
   async overridePermissions(
     origin: string,
-    permissions: Protocol.Browser.PermissionType[]
```

Comment:
I think this was a bug in the typings. With this type, we can only override with permissions where `web` and `protocol` permission names coincide?